src/gui/entry/EntryModel.cpp

```diff
@@ -318,6 +360,27 @@ QVariant EntryModel::data(const QModelIndex& index, int role) const
         if (index.column() == Paperclip) {
             return Qt::AlignCenter;
```

Comment:
We should be center aligning the password strength column as well.

Reply:
In order to fix the alignment, I needed to override the QStyledItemDelegate's paint function with my own as the alignment role in qt does not apply to decorations to a cell. I currently have the implementation in the entry view files at the moment and am having the color take up the entire cell instead of centering the decoration as I thought that this may have been a more clear implementation. The color will show through selection as well. I put an example below:
![Capture](https://user-images.githubusercontent.com/20828023/103396802-34567b00-4b03-11eb-9186-0617f3bc9fa2.PNG)